meeting-minutes/2024-12-18.md

```diff
@@ -0,0 +1,134 @@
+# 1. Opening Activities
+
+## 1.1 Opening comments (Co-Chair)
+
+## 1.2 Introduction of participants/roll call (Co-Chair)
+
+Quorum requires participation of 9 or more of the 17 voting members (including the officers).
+
+| First Name | Last Name  | Company                                                     | Role(s)                   | Present |
+|:-----------|:-----------|:------------------------------------------------------------|:--------------------------|:--------|
+| Adrian     | Diglio     | Microsoft                                                   | Voting Member             | ??      |
+| David      | Kemp       | National Security Agency                                    | Member                    | ??      |
+| Denny      | Page       | Individual                                                  | Voting Member             | ???     |
+| Duncan     | Sparrell   | sFractal Consulting LLC                                     | Voting Member             | ??      |
+| Feng       | Cao        | Oracle                                                      | Member                    | ???     |
+| Harin      | Sarda      | Cisco Systems                                               | Voting Member             | ???     |
+| Jautau     | White      | Microsoft                                                   | Voting Member             | ???     |
+| Jeremy     | Rickard    | Microsoft                                                   | Member                    | ??      |
+| Justin     | Murphy     | DHS Cybersecurity and Infrastructure Security Agency (CISA) | Co-Chair                  | ???     |
+| Kris       | Vandecruys | Cisco Systems                                               | Voting Member             | ???     |
+| Kunal      | Modasiya   | Qualys, Inc.                                                | Member                    | ??      |
+| Langley    | Rock       | Dell                                                        | Voting Member             | ???     |
+| Martin     | Prpic      | Red Hat                                                     | Voting Member             | ??      |
+| Omar       | Santos     | Cisco Systems                                               | Co-Chair                  | ???     |
+| Pablo      | Quiroga    | Qualys, Inc.                                                | Voting Member             | ???     |
+| Peter      | Gephardt   | IBM                                                         | Member                    | ??      |
+| Przemyslaw | Roguski    | Red Hat                                                     | Voting Member             | ???     |
+| Shridhar   | Chari      | Cisco Systems                                               | Member                    | ??      |
```

Comment:
Indeterminable whether Shridar has gained voting rights.

Reply:
Yes, we need to get the September meeting minutes to determine Shridar's status.
Not present in August, status Member
September Unknown
Present in October. If present in September, would have regained voting rights.
Not present in November.
If not present by end of meeting in December, would lose voting rights.